controllers/component/component_controller_test.go

```diff
@@ -22,7 +23,7 @@ import (
 	klog "k8s.io/klog/v2"
 )
 
-var _ = Describe("ComponentController", func() {
+var _ = Describe("ComponentController", Ordered, func() {
```

Comment:
Why do we need this change to ordered tests?

Reply:
Unit test won't pass if its not ordered, same issue as we had in 841 with removing resoures.